.github/workflows/cd-release.yaml

```diff
@@ -97,6 +97,15 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version-file: go.mod
+      - name: Free up disk space
```

Comment:
Should add a TODO about hosting our own runners, which seems inevitable at some point